docs/tutorials/tutorials-runtime-upgrade.md

```diff
@@ -38,6 +36,7 @@ sidebar_label: Runtime Upgrade
 
 * Click "Submit Transaction" and then "Sign and Submit".
 
+![Submit Preimage](/img/tutorial/submit_preimage.png "Submit Preimage")
```

Comment:
i think it's worth mentioning that if they forgot to copy the preimage has, they can easily regenerate and copy it again by choosing the same extrinsic (but just not submitting it again)